### ISSUE TITLE
cs practice and implementation

### ISSUE THREAD

**Issue by BrooksPatton** [BrooksPatton/project-ideas#29]
A website where we can practice implementing cs algorithm's and data structures. It can have a repl attached to it through docker or some other vm.

**Comment by lc00** [comment]
this is like coderbyte or web app similar to it, maaybe it's a good idea

**Comment by BrooksPatton** [comment]
yeah, I'm thinking like coderbyte but only for cs algorithms and data structures
